literature/literature.bib

```diff
@@ -228,6 +228,16 @@ @inproceedings{SBK+:VaMoS25
 	keywords  = {software product lines, t-wise sampling},
 }
 
+@inproceedings{SKGT:WSRE25,
+	author    = {Sundermann, Rahel and Krieter, Sebastian and Glimm, Birte and Th{\"{u}}m, Thomas},
+	title     = {{Teaching Software Quality with Adaptive Source Code Feedback}},
+	booktitle = wsre,
+	note      = toappear,
+	publisher = gi,
+	year      = 2025,
+	keywords  = {software quality, adaptive learning},
+}
```

Comment:
address missing

Reply:
But maybe its okay to add that once the paper is published. Either way it should not be forgotten. Same for `month`.